src/main/java/it/gov/pagopa/pu/organization/mapper/OrgSilServiceMapper.java

```diff
@@ -0,0 +1,49 @@
+package it.gov.pagopa.pu.organization.mapper;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import it.gov.pagopa.pu.organization.dto.generated.OrgSilServiceDTO;
+import it.gov.pagopa.pu.organization.model.OrgSilService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class OrgSilServiceMapper {
+
+  private final ObjectMapper objectMapper;
+
+  public OrgSilServiceDTO fromEntity(OrgSilService orgSilService) {
+    if (orgSilService == null) {
+      return null;
+    }
+
+    OrgSilServiceDTO dto = new OrgSilServiceDTO();
+    dto.setOrgSilServiceId(orgSilService.getOrgSilServiceId());
+    dto.setOrganizationId(orgSilService.getOrganizationId());
+    dto.setServiceType(orgSilService.getServiceType());
+    dto.setServiceUrl(orgSilService.getServiceUrl());
+    dto.setApplicationName(orgSilService.getApplicationName());
+    dto.setFlagLegacy(orgSilService.isFlagLegacy());
+    dto.setAuthConfig(orgSilService.getAuthConfig());
```

Comment:
we cannot do this...
DTO's authConfig and entity's authConfig shouldn't be the same type!

DTO's authConfig should contain plain text for secrets (String), while entity's authConfig contain cipher value (byte[])

Reply:
ps: if you want, you could rename entity's authConfig using suffix `Ciphered` to make explicit the difference between classes